src/api-docs/source/includes/_wallet_rpc.md

```diff
@@ -2246,8 +2252,78 @@ N. | Name | Default |  Description
 6 | subtractfee | Optional | (bool) Subtract the transaction fee equally from the output amounts
 
 
+## createsendtoaddress
+<aside class="warning">
+This command involves entering HNS values, be careful with <a href="#values">different formats</a> of values for different APIs.
+</aside>
+
+```javascript
+let address, amount, comment, comment_to, subtractFee;
+```
+
+```shell--vars
+address='rs1qkk62444euknkya4qws9cg3ej3au24n635n4qye'
+amount=1.010101
+comment="this_is_ignored"
+comment_to="this_is_ignored"
+subtractfee=true
+```
+
+```shell--curl
+curl http://x:api-key@127.0.0.1:14039 \
+  -X POST \
+  --data '{
+    "method": "createsendtoaddress",
+    "params": [ "'$address'", '$amount', "'$comment'", "'$comment_to'", '$subtractfee' ]
+  }'
+```
+
+```shell--cli
+hsw-cli rpc createsendtoaddress $address $amount $comment $commnt_to $subtractfee
+```
+
+```javascript
+const {WalletClient} = require('hs-client');
+const {Network} = require('hsd');
+const network = Network.get('regtest');
+
+const walletOptions = {
+  port: network.walletPort,
+  apiKey: 'api-key'
+}
+
+const walletClient = new WalletClient(walletOptions);
+
+(async () => {
+  const result = await walletClient.execute('createsendtoaddress', [address, amount, comment, comment_to, subtractfee]);
+  console.log(result);
+})();
+```
+
+> The above command returns JSON "result" like this:
+
+```
+552006b288266ab26fa30d9048b758a469a4101fd8235eff2384141ca5cf604d
```

Comment:
That's all this RPC call returns? When I run the example I get a JSON of the entire TX.

Reply:
Added full response.